src/App.js

```diff
@@ -1,25 +1,34 @@
-import logo from './logo.svg';
-import './App.css';
+import background from '../src/assets/viary-bg.jpg'
+import Header from '../src/components/Header'
+import MemoriesPage from './pages/MemoriesPage'
+import image from './assets/images/memory_1.jpg'
```

Comment:
The variable `image` is not very expressive.

```suggestion
import memory1 from './assets/images/memory_1.jpg'
```
or
```suggestion
import mountainsImage from './assets/images/mountains.jpg'
```

Reply:
Change image-names in "memory1" etc. and replace image with memory1 (temporary solution)